Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ualibraries/ruby_plus2.3.3
+# https://github.com/ualbertalib/DIDockerImages/blob/master/RubyPlus/Dockerfile
+RUN apt-get update -qq && \
+    apt-get install -y libpq-dev \
+                       nodejs \
+                       libreoffice-core \
+                       libreoffice-base-core \
+                       libreoffice-common \
+                       imagemagick \
+                       unzip && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /app
+WORKDIR /app
+ADD . /app
+RUN echo "gem 'tinymce-rails-imageupload', github: 'PerfectlyNormal/tinymce-rails-imageupload'" >> /app/Gemfile
+RUN cd /app && bundle install
+RUN cd /app && rake assets:precompile
+ADD bin/start_hn2.sh /usr/local/bin
+
+EXPOSE 3000
```

Comment:
Yes we need to expose it inside docker image.  This instruction means that other containers can connect to this one to the port 3000. This is what Nginx will do - will connect to instance of hydranorth2 running container and redirect port 3000 to its own port 80 which in turn will be redirected to atomic host port pointed by HOSTPORT environment variable.